libs/langchain/langchain/vectorstores/cratedb/base.py

```diff
@@ -168,9 +168,11 @@ def add_embeddings(
             self.delete_collection()
         self.create_tables_if_not_exists()
         self.create_collection()
-        return super().add_embeddings(
+        embedding_ids = super().add_embeddings(
             texts=texts, embeddings=embeddings, metadatas=metadatas, ids=ids, **kwargs
         )
+        refresh_table(self.Session(), self.EmbeddingStore)
```

Comment:
This is irrelevant to the bulk, it was missing anyway, right?

Reply:
Thanks for asking. Yes, this synchronization operation was missing anyway, but it made the flaw more appearant: Making the insert operation faster, made the corresponding tests trip right away.

Reply:
thx!